src/amp/components/Ticker.tsx

```diff
@@ -0,0 +1,94 @@
+import {css} from 'emotion';
+import {headline} from '@guardian/src-foundations/typography';
+import {neutral, brandAlt} from '@guardian/src-foundations/palette';
+import React from 'react';
+
+const tickerWrapper = css`
+    margin-bottom: 20px;
+`;
+const left = css`
+    text-align: left;
+    display: flex;
+    flex-wrap: nowrap;
+    flex-direction: column;
+    justify-content: flex-end;
+`;
+const right = css`
+    text-align: right;
+    display: flex;
+    flex-wrap: nowrap;
+    flex-direction: column;
+    justify-content: flex-end;
+`;
+const tickerInfo = css`
```

Comment:
as a convention we are trying to add the suffix `Styles` to const related to styling. Thus we could name this variable `tickerInfoStyle`, this should also apply to other style consts.

Reply:
✅